metals/src/main/scala/scala/meta/internal/metals/Cancelable.scala

```diff
@@ -21,6 +21,11 @@ object Cancelable {
   def cancelEach[T](iterable: Iterable[T])(fn: T => Unit): Unit = {
     cancelAll(iterable.map(elem => Cancelable(() => fn(elem))))
   }
+
+  def cancelAll(cancelables: Cancelable*): Unit = {
```

Comment:
nit: can we remove this overload in favor of explicitly wrapping call-site arguments in a `List(..)`? I generally try to avoid varargs where possible.